docs/get-started.md

```diff
@@ -19,7 +19,7 @@ environment.
 ## Process overview
 
 * [Step 1: Create a React Native project](#step-1-create-a-react-native-project)
-* [Step 2: Install and link the React Native plugin for In-App Payments SDK](#step-2-install-and-link-the-react-native-plugin-for-in-app-payments-sdk)
+* [Step 2: Install the React Native plugin for In-App Payments SDK](#step-2-install-and-link-the-react-native-plugin-for-in-app-payments-sdk)
```

Comment:
I think this link doesn't work anymore.  

Should it be `#step-2-install-the-react-native-plugin-for-in-app-payments-sdk`?